apps/mongodb/workload/run_litmus_test.yml

```diff
@@ -23,17 +23,14 @@ spec:
             #value: log_plays
             value: default
 
-          - name: POD_NAME
-            value: mongo-0
-
-          - name: SERVICE_NAME 
-            value: mongo
-
           - name: LOADGEN_LABEL
             value: loadgen=mongo-loadgen
```

Comment:
@shashank855  can you place the label value within the quote  ('loadgen=mongo-loadgen')